python/plugins/db_manager/db_plugins/postgis/sql_dictionary.py

```diff
@@ -174,6 +174,29 @@
                        "ST_ZMax", "ST_ZMin",
                        # 7.13. Exceptional Functions
                        "*postgis_addbbox", "*postgis_dropbbox", "*postgis_hasbbox"
+                       # Raster functions
+                       "AddRasterConstraints", "DropRasterConstraints", "AddOverviewConstraints", "DropOverviewConstraints",
```

Comment:
Just noticed this -- you're missing a comma after "postgis_hasbbox" on the previous line

Reply:
Ouch. Just pushed fix to master. Will issue PR for 3.10 and 3.12